src/omero_cli_render.py

```diff
@@ -67,29 +75,33 @@
       <index>:
         ...
     greyscale: <(bool)>
-
-    # Omitted fields will keep their current values, omitted channel-indices
-    # will be turned off.
-
-    bin/omero render set --copy Screen:1 <YAML or JSON file>
-    # Equivalent to calling set for a single image followed by calling copy
-    # for all the remaining images in the collection, using the first image
-    # as the source. Note using this flag may have different results from not
-    # using it if the images had different settings to begin with and you are
-    # only overriding a subset of the settings (all images will end up with
-    # the same full rendering settings).
-
-    bin/omero render set --skipthumbs ...
-    # Update rendering settings but don't regenerate thumbnails.
-
-    # %(COPY)s
-    bin/omero render copy RenderingDef:1 Image:123
-    bin/omero render copy Image:456: Image:789
-    bin/omero render copy Image:456: Image:222 Image:333
-    bin/omero render copy Image:456: Plate:1
-    bin/omero render copy Image:456: Screen:2
-    bin/omero render copy Image:456: Dataset:3
-""" % DESC
+    
+    For example:
+    channels:
+      1:
+        color: "FF0000"
+        label: "Red"
+        min: 1
+        max: 255
+        active: True
+      2:
+        color: "00FF00"
+      ...
+      
+    # Omitted fields will keep their current values
+    
+    # If a channel is turned off (active: False) other settings like min,
+    # max, etc. will be ignored.
```

Comment:
``If the file specifies to turn of a channel (active: False) then the other settings like min, max, etc. which might be specified for that channel in the same file will be ignored.``

Is that maybe clearer ?